from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(476, 613)
        MainWindow.setMinimumSize(QtCore.QSize(476, 500))
        MainWindow.setMaximumSize(QtCore.QSize(600, 700))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(":/icons/icons/folder (1).png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        MainWindow.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setContentsMargins(0, 0, 0, 0)
        self.gridLayout.setSpacing(0)
        self.gridLayout.setObjectName("gridLayout")
        self.tabWidget = QtWidgets.QTabWidget(self.centralwidget)
        self.tabWidget.setCursor(QtGui.QCursor(QtCore.Qt.CursorShape.ArrowCursor))
        self.tabWidget.setFocusPolicy(QtCore.Qt.FocusPolicy.NoFocus)
        self.tabWidget.setContextMenuPolicy(QtCore.Qt.ContextMenuPolicy.DefaultContextMenu)
        self.tabWidget.setToolTip("")
        self.tabWidget.setIconSize(QtCore.QSize(60, 25))
        self.tabWidget.setElideMode(QtCore.Qt.TextElideMode.ElideLeft)
        self.tabWidget.setDocumentMode(False)
        self.tabWidget.setTabsClosable(False)
        self.tabWidget.setMovable(True)
        self.tabWidget.setTabBarAutoHide(False)
        self.tabWidget.setObjectName("tabWidget")
        self.tab = QtWidgets.QWidget()
        self.tab.setObjectName("tab")
        self.gridLayout_2 = QtWidgets.QGridLayout(self.tab)
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout()
        self.verticalLayout_2.setContentsMargins(100, 23, 100, 15)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.combo_lbl = QtWidgets.QLabel(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.combo_lbl.sizePolicy().hasHeightForWidth())
        self.combo_lbl.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.combo_lbl.setFont(font)
        self.combo_lbl.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.combo_lbl.setObjectName("combo_lbl")
        self.verticalLayout_2.addWidget(self.combo_lbl)
        self.horizontalLayout_2 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_2.setObjectName("horizontalLayout_2")
        self.select_radio_btn = QtWidgets.QRadioButton(self.tab)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.select_radio_btn.setFont(font)
        self.select_radio_btn.setCursor(QtGui.QCursor(QtCore.Qt.CursorShape.PointingHandCursor))
        self.select_radio_btn.setChecked(True)
        self.select_radio_btn.setObjectName("select_radio_btn")
        self.horizontalLayout_2.addWidget(self.select_radio_btn)
        self.includ_radio_btn = QtWidgets.QRadioButton(self.tab)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.includ_radio_btn.setFont(font)
        self.includ_radio_btn.setCursor(QtGui.QCursor(QtCore.Qt.CursorShape.PointingHandCursor))
        self.includ_radio_btn.setObjectName("includ_radio_btn")
        self.horizontalLayout_2.addWidget(self.includ_radio_btn)
        self.verticalLayout_2.addLayout(self.horizontalLayout_2)
        self.horizontalLayout_4 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_4.setObjectName("horizontalLayout_4")
        self.combo = QtWidgets.QComboBox(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.combo.sizePolicy().hasHeightForWidth())
        self.combo.setSizePolicy(sizePolicy)
        self.combo.setObjectName("combo")
        self.horizontalLayout_4.addWidget(self.combo)
        self.include_line_edit = QtWidgets.QLineEdit(self.tab)
        self.include_line_edit.setEnabled(False)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.include_line_edit.sizePolicy().hasHeightForWidth())
        self.include_line_edit.setSizePolicy(sizePolicy)
        self.include_line_edit.setObjectName("include_line_edit")
        self.horizontalLayout_4.addWidget(self.include_line_edit)
        self.verticalLayout_2.addLayout(self.horizontalLayout_4)
        spacerItem = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout_2.addItem(spacerItem)
        self.open_up_lbl = QtWidgets.QLabel(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.open_up_lbl.sizePolicy().hasHeightForWidth())
        self.open_up_lbl.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.open_up_lbl.setFont(font)
        self.open_up_lbl.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.open_up_lbl.setObjectName("open_up_lbl")
        self.verticalLayout_2.addWidget(self.open_up_lbl)
        self.horizontalLayout_10 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_10.setSizeConstraint(QtWidgets.QLayout.SizeConstraint.SetFixedSize)
        self.horizontalLayout_10.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout_10.setSpacing(6)
        self.horizontalLayout_10.setObjectName("horizontalLayout_10")
        self.open_btn = QtWidgets.QPushButton(self.tab)
        font = QtGui.QFont()
        font.setFamily("Segoe UI")
        font.setBold(False)
        self.open_btn.setFont(font)
        self.open_btn.setObjectName("open_btn")
        self.horizontalLayout_10.addWidget(self.open_btn)
        self.open_lbl = QtWidgets.QLabel(self.tab)
        self.open_lbl.setObjectName("open_lbl")
        self.horizontalLayout_10.addWidget(self.open_lbl)
        self.verticalLayout_2.addLayout(self.horizontalLayout_10)
        spacerItem1 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout_2.addItem(spacerItem1)
        self.exception_folder_up_lbl = QtWidgets.QLabel(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.exception_folder_up_lbl.sizePolicy().hasHeightForWidth())
        self.exception_folder_up_lbl.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.exception_folder_up_lbl.setFont(font)
        self.exception_folder_up_lbl.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.exception_folder_up_lbl.setObjectName("exception_folder_up_lbl")
        self.verticalLayout_2.addWidget(self.exception_folder_up_lbl)
        self.horizontalLayout_11 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_11.setObjectName("horizontalLayout_11")
        self.exception_folder_btn = QtWidgets.QPushButton(self.tab)
        self.exception_folder_btn.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.exception_folder_btn.setFont(font)
        self.exception_folder_btn.setObjectName("exception_folder_btn")
        self.horizontalLayout_11.addWidget(self.exception_folder_btn)
        self.exception_folder_lbl = QtWidgets.QLabel(self.tab)
        self.exception_folder_lbl.setObjectName("exception_folder_lbl")
        self.horizontalLayout_11.addWidget(self.exception_folder_lbl)
        self.verticalLayout_2.addLayout(self.horizontalLayout_11)
        spacerItem2 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout_2.addItem(spacerItem2)
        self.exception_file_up_lbl = QtWidgets.QLabel(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.exception_file_up_lbl.sizePolicy().hasHeightForWidth())
        self.exception_file_up_lbl.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.exception_file_up_lbl.setFont(font)
        self.exception_file_up_lbl.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.exception_file_up_lbl.setObjectName("exception_file_up_lbl")
        self.verticalLayout_2.addWidget(self.exception_file_up_lbl)
        self.horizontalLayout_12 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_12.setObjectName("horizontalLayout_12")
        self.exception_file_btn = QtWidgets.QPushButton(self.tab)
        self.exception_file_btn.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.exception_file_btn.setFont(font)
        self.exception_file_btn.setObjectName("exception_file_btn")
        self.horizontalLayout_12.addWidget(self.exception_file_btn)
        self.exception_file_lbl = QtWidgets.QLabel(self.tab)
        self.exception_file_lbl.setObjectName("exception_file_lbl")
        self.horizontalLayout_12.addWidget(self.exception_file_lbl)
        self.verticalLayout_2.addLayout(self.horizontalLayout_12)
        spacerItem3 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout_2.addItem(spacerItem3)
        self.create_file_up_lbl = QtWidgets.QLabel(self.tab)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.create_file_up_lbl.sizePolicy().hasHeightForWidth())
        self.create_file_up_lbl.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.create_file_up_lbl.setFont(font)
        self.create_file_up_lbl.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.create_file_up_lbl.setObjectName("create_file_up_lbl")
        self.verticalLayout_2.addWidget(self.create_file_up_lbl)
        self.horizontalLayout_13 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_13.setObjectName("horizontalLayout_13")
        self.create_btn = QtWidgets.QPushButton(self.tab)
        self.create_btn.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.create_btn.setFont(font)
        self.create_btn.setObjectName("create_btn")
        self.horizontalLayout_13.addWidget(self.create_btn)
        self.create_file_lbl = QtWidgets.QLabel(self.tab)
        self.create_file_lbl.setObjectName("create_file_lbl")
        self.horizontalLayout_13.addWidget(self.create_file_lbl)
        self.verticalLayout_2.addLayout(self.horizontalLayout_13)
        spacerItem4 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout_2.addItem(spacerItem4)
        self.horizontalLayout_15 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_15.setObjectName("horizontalLayout_15")
        self.clear_btn = QtWidgets.QPushButton(self.tab)
        font = QtGui.QFont()
        font.setBold(False)
        self.clear_btn.setFont(font)
        self.clear_btn.setObjectName("clear_btn")
        self.horizontalLayout_15.addWidget(self.clear_btn)
        self.search_btn = QtWidgets.QPushButton(self.tab)
        font = QtGui.QFont()
        font.setBold(False)
        self.search_btn.setFont(font)
        self.search_btn.setObjectName("search_btn")
        self.horizontalLayout_15.addWidget(self.search_btn)
        self.verticalLayout_2.addLayout(self.horizontalLayout_15)
        self.copy_btn = QtWidgets.QPushButton(self.tab)
        self.copy_btn.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.copy_btn.setFont(font)
        self.copy_btn.setObjectName("copy_btn")
        self.verticalLayout_2.addWidget(self.copy_btn)
        self.delete_btn = QtWidgets.QPushButton(self.tab)
        self.delete_btn.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.delete_btn.setFont(font)
        self.delete_btn.setObjectName("delete_btn")
        self.verticalLayout_2.addWidget(self.delete_btn)
        self.gridLayout_2.addLayout(self.verticalLayout_2, 0, 0, 1, 1)
        icon1 = QtGui.QIcon()
        icon1.addPixmap(QtGui.QPixmap(":/icons/icons/folder (2).png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        self.tabWidget.addTab(self.tab, icon1, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.gridLayout_5 = QtWidgets.QGridLayout(self.tab_2)
        self.gridLayout_5.setObjectName("gridLayout_5")
        self.verticalLayout = QtWidgets.QVBoxLayout()
        self.verticalLayout.setContentsMargins(100, 0, 100, 134)
        self.verticalLayout.setSpacing(6)
        self.verticalLayout.setObjectName("verticalLayout")
        spacerItem5 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem5)
        self.label = QtWidgets.QLabel(self.tab_2)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.label.sizePolicy().hasHeightForWidth())
        self.label.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label.setFont(font)
        self.label.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.label.setObjectName("label")
        self.verticalLayout.addWidget(self.label)
        self.lineEdit = QtWidgets.QLineEdit(self.tab_2)
        self.lineEdit.setText("")
        self.lineEdit.setObjectName("lineEdit")
        self.verticalLayout.addWidget(self.lineEdit)
        spacerItem6 = QtWidgets.QSpacerItem(20, 47, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem6)
        self.checkBox = QtWidgets.QCheckBox(self.tab_2)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.checkBox.setFont(font)
        self.checkBox.setObjectName("checkBox")
        self.verticalLayout.addWidget(self.checkBox)
        self.checkBox_2 = QtWidgets.QCheckBox(self.tab_2)
        self.checkBox_2.setObjectName("checkBox_2")
        self.verticalLayout.addWidget(self.checkBox_2)
        spacerItem7 = QtWidgets.QSpacerItem(20, 46, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem7)
        self.label_2 = QtWidgets.QLabel(self.tab_2)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.label_2.sizePolicy().hasHeightForWidth())
        self.label_2.setSizePolicy(sizePolicy)
        font = QtGui.QFont()
        font.setPointSize(10)
        self.label_2.setFont(font)
        self.label_2.setAlignment(QtCore.Qt.AlignmentFlag.AlignCenter)
        self.label_2.setObjectName("label_2")
        self.verticalLayout.addWidget(self.label_2)
        self.horizontalLayout = QtWidgets.QHBoxLayout()
        self.horizontalLayout.setSizeConstraint(QtWidgets.QLayout.SizeConstraint.SetFixedSize)
        self.horizontalLayout.setContentsMargins(-1, 0, -1, -1)
        self.horizontalLayout.setSpacing(6)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.pushButton = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setBold(False)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.horizontalLayout.addWidget(self.pushButton)
        self.label_3 = QtWidgets.QLabel(self.tab_2)
        sizePolicy = QtWidgets.QSizePolicy(QtWidgets.QSizePolicy.Policy.Preferred, QtWidgets.QSizePolicy.Policy.Fixed)
        sizePolicy.setHorizontalStretch(0)
        sizePolicy.setVerticalStretch(0)
        sizePolicy.setHeightForWidth(self.label_3.sizePolicy().hasHeightForWidth())
        self.label_3.setSizePolicy(sizePolicy)
        self.label_3.setObjectName("label_3")
        self.horizontalLayout.addWidget(self.label_3)
        self.verticalLayout.addLayout(self.horizontalLayout)
        spacerItem8 = QtWidgets.QSpacerItem(20, 46, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem8)
        self.horizontalLayout_3 = QtWidgets.QHBoxLayout()
        self.horizontalLayout_3.setSpacing(0)
        self.horizontalLayout_3.setObjectName("horizontalLayout_3")
        self.pushButton_4 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setBold(False)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setObjectName("pushButton_4")
        self.horizontalLayout_3.addWidget(self.pushButton_4)
        self.pushButton_3 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setBold(False)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout_3.addWidget(self.pushButton_3)
        self.verticalLayout.addLayout(self.horizontalLayout_3)
        self.pushButton_6 = QtWidgets.QPushButton(self.tab_2)
        self.pushButton_6.setEnabled(False)
        font = QtGui.QFont()
        font.setBold(False)
        self.pushButton_6.setFont(font)
        self.pushButton_6.setObjectName("pushButton_6")
        self.verticalLayout.addWidget(self.pushButton_6)
        spacerItem9 = QtWidgets.QSpacerItem(20, 40, QtWidgets.QSizePolicy.Policy.Minimum, QtWidgets.QSizePolicy.Policy.Expanding)
        self.verticalLayout.addItem(spacerItem9)
        self.lineEdit_2 = QtWidgets.QLineEdit(self.tab_2)
        self.lineEdit_2.setObjectName("lineEdit_2")
        self.verticalLayout.addWidget(self.lineEdit_2)
        self.pushButton_2 = QtWidgets.QPushButton(self.tab_2)
        font = QtGui.QFont()
        font.setBold(False)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        self.verticalLayout.addWidget(self.pushButton_2)
        self.gridLayout_5.addLayout(self.verticalLayout, 0, 0, 1, 1)
        icon2 = QtGui.QIcon()
        icon2.addPixmap(QtGui.QPixmap(":/icons/icons/document.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        self.tabWidget.addTab(self.tab_2, icon2, "")
        self.gridLayout.addWidget(self.tabWidget, 0, 0, 1, 1)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 476, 22))
        self.menubar.setObjectName("menubar")
        self.menuMen = QtWidgets.QMenu(self.menubar)
        self.menuMen.setObjectName("menuMen")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionGitHub = QtGui.QAction(MainWindow)
        icon3 = QtGui.QIcon()
        icon3.addPixmap(QtGui.QPixmap(":/icons/icons/Papirus-Team-Papirus-Apps-Github.ico"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        self.actionGitHub.setIcon(icon3)
        self.actionGitHub.setObjectName("actionGitHub")
        self.actionHaqq_nda = QtGui.QAction(MainWindow)
        icon4 = QtGui.QIcon()
        icon4.addPixmap(QtGui.QPixmap(":/icons/icons/info.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        self.actionHaqq_nda.setIcon(icon4)
        self.actionHaqq_nda.setObjectName("actionHaqq_nda")
        self.menuMen.addAction(self.actionGitHub)
        self.menuMen.addAction(self.actionHaqq_nda)
        self.menubar.addAction(self.menuMen.menuAction())

        self.retranslateUi(MainWindow)
        self.tabWidget.setCurrentIndex(0)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.combo_lbl.setText(_translate("MainWindow", "Kopyalanacaq fayl tipini seçin"))
        self.select_radio_btn.setText(_translate("MainWindow", "Seçəcəm"))
        self.includ_radio_btn.setText(_translate("MainWindow", "Daxil edəcəm"))
        self.open_up_lbl.setText(_translate("MainWindow", "Axtarılacaq qovluğu seçin"))
        self.open_btn.setText(_translate("MainWindow", "Seç"))
        self.open_lbl.setText(_translate("MainWindow", "Seçilməyib"))
        self.exception_folder_up_lbl.setText(_translate("MainWindow", "Axtarılmayacaq qovluqlar seçin (istəyə bağlı)"))
        self.exception_folder_btn.setText(_translate("MainWindow", "Seç"))
        self.exception_folder_lbl.setText(_translate("MainWindow", "Seçilməyib"))
        self.exception_file_up_lbl.setText(_translate("MainWindow", "İstisna fayllar seçin (istəyə bağlı)"))
        self.exception_file_btn.setText(_translate("MainWindow", "Seç"))
        self.exception_file_lbl.setText(_translate("MainWindow", "Seçilməyib"))
        self.create_file_up_lbl.setText(_translate("MainWindow", "Faylların kopyalanacağı yeri seçin"))
        self.create_btn.setText(_translate("MainWindow", "Seç"))
        self.create_file_lbl.setText(_translate("MainWindow", "Seçilməyib"))
        self.clear_btn.setText(_translate("MainWindow", "Təmizlə"))
        self.search_btn.setText(_translate("MainWindow", "Axtar"))
        self.copy_btn.setText(_translate("MainWindow", "Kopyala"))
        self.delete_btn.setText(_translate("MainWindow", "Sil"))
        self.tabWidget.setTabToolTip(self.tabWidget.indexOf(self.tab), _translate("MainWindow", "Faylları kopyala, sil"))
        self.label.setText(_translate("MainWindow", "Fayl adı daxil edin"))
        self.lineEdit.setPlaceholderText(_translate("MainWindow", "Fayl adı"))
        self.checkBox.setText(_translate("MainWindow", "Böyük (kiçik) hərf fərqi"))
        self.checkBox_2.setText(_translate("MainWindow", "Fayl uzantısını diqqətə alma"))
        self.label_2.setText(_translate("MainWindow", "Axtarılacaq qovluğu seçin"))
        self.pushButton.setText(_translate("MainWindow", "Seç"))
        self.label_3.setText(_translate("MainWindow", "Seçilməyib"))
        self.pushButton_4.setText(_translate("MainWindow", "Təmizlə"))
        self.pushButton_3.setText(_translate("MainWindow", "Axtar"))
        self.pushButton_6.setText(_translate("MainWindow", "Sil"))
        self.lineEdit_2.setPlaceholderText(_translate("MainWindow", "Fayl, qovluq adresi"))
        self.pushButton_2.setText(_translate("MainWindow", "Get"))
        self.tabWidget.setTabToolTip(self.tabWidget.indexOf(self.tab_2), _translate("MainWindow", "Fayl tap, sil"))
        self.menuMen.setTitle(_translate("MainWindow", "Menü"))
        self.actionGitHub.setText(_translate("MainWindow", "GitHub"))
        self.actionHaqq_nda.setText(_translate("MainWindow", "Haqqında"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    MainWindow = QtWidgets.QMainWindow()
    ui = Ui_MainWindow()
    ui.setupUi(MainWindow)
    MainWindow.show()
    sys.exit(app.exec())
